doc/release-notes-5588.md

```diff
@@ -0,0 +1,14 @@
+Block Reward Reallocation
+-------------------------
+
+Once Masternode Reward Location Reallocation activates:
+
+- Treasury is bumped to 20% of block subsidy.
+- Block reward shares are immediately set to 75% for MN and 25% miners. MN reward share should be 75% of block reward in order to represent 60% of the block subsidy. (according to the proposal).
```

Comment:
Do you think it's important to mention the 75%/25% split of the 80% piece? Or could we simply mention the overall percentage of 20/60/20?

Reply:
When reviewing this PR it helps people understand the actual changes, but maybe you are right: we can drop it as it is too much detailed for v20 release notes.